doc/manual/preprocessing/ssp.rst

```diff
@@ -67,14 +49,20 @@ conditions described above are satisfied. We can now construct the
 orthogonal complement operator
 
 .. math::    P_{\perp} = I - UU^T
+   :label: projector
+
+and apply it to :math:`b(t)` in Equation :eq:`additive_model` yielding
+
+.. math::    b_{s}(t) \approx P_{\perp}b(t)\ ,
+   :label: result
 
-and apply it to :math:`b(t)` yielding
+since :math:`P_{\perp}b_n(t) = P_{\perp}(Uc_n(t) + e(t)) \approx 0` and :math:`P_{\perp}b_{s}(t) \approx b_{s}(t)`. The projection operator :math:`P_{\perp}` is
+called the **signal-space projection operator**.
 
-.. math::    b(t) = P_{\perp}b_s(t)\ ,
```

Comment:
Somehow, this looked incorrect to me. It is there in the MNE manual too but I can't follow the steps ...